src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -337,6 +377,17 @@ public String getHeaderField(String name) {
         return connection.getHeaderField(name);
     }
 
+    /**
+     * Returns the response header with the given name.
+     *
+     * @param name        the name of the header to fetch
+     * @param defaultDate a default value to return if the header field is missing or malformed.
+     * @return the value of the given header in the response or the given <tt>defaultDate</tt>
+     */
+    public long getHeaderFieldDate(String name, long defaultDate) {
```

Comment:
maybe getHeaderAsTimestamp or the like? as we do not return a date but (presumably an UTC timestamp)

Reply:
oooor return an Optional<LocalDateTime> or the like